torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1287,6 +1287,20 @@ that adds a prefix to each key inserted to the store.
       py::arg("tensor_indices") = std::vector<int64_t>(),
       py::call_guard<py::gil_scoped_release>());
 
+  module.def(
+      "_verify_replicas_within_process",
+      &::c10d::verify_replicas_within_process,
+      py::arg("replicas"),
+      py::arg("expect_sparse_gradient"),
+      py::call_guard<py::gil_scoped_release>());
+
+  module.def(
+      "_verify_replica0_across_processes",
```

Comment:
nit: it can be renamed as "_verify_model_across_ranks", so it does not need to define a new function in python again